.cache/cache-refresh-daemonset.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: springtrader-cache-refresh
+  labels:
+    k8s-app: springtrader-cache-refresh
+spec:
+  selector:
+    matchLabels:
+      name: springtrader-cache-refresh
+  template:
+    metadata:
+      labels:
+        name: springtrader-cache-refresh
+    spec:
+      containers:
+      - name: springtrader-cache-refresh
+        image: artifactory.toolchain.lead.prod.liatr.io/docker-registry/flywheel/builder-image-skaffold:v1.1.1-1-g6b3c49e
+        command: ["/bin/sh"]
+        args: ["-c", "while true; do docker pull artifactory.toolchain.lead.prod.liatr.io/docker-registry/liatrio/springtrader-builder:latest; docker pull artifactory.toolchain.lead.prod.liatr.io/docker-registry/liatrio/springtrader-runner:latest; sleep 600; done"]
+        volumeMounts:
+        - mountPath: /var/run/docker.sock
+          name: docker
+      volumes:
```

Comment:
I think `/var/lib/docker` needs to be mounted too.

Reply:
I used the same mounts that the jenkins agent containers use (which do not include /var/lib/docker). My guess is that all of the container interactions go through the daemon and do not access the docker filesystem directly so it's not needed but I could be wrong. Do you know for sure they need the /var/lib/docker?

Reply:
No, it was just a guess.  If it's working then its probably not needed